RevenueCatUI/Data/Localization.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  Localization.swift
+//
+//
+//  Created by Nacho Soto on 7/20/23.
+//
+
+import Foundation
+import RevenueCat
+
+enum Localization {
+
+    /// - Returns: an appropriately short abbreviation for the given `unit`.
+    static func abbreviatedUnitLocalizedString(
```

Comment:
See unit tests for examples.

Reply:
it's awesome that we can provide this part for free